go/vt/tabletserver/splitquery/full_scan_algorithm_test.go

```diff
@@ -29,7 +29,7 @@ func TestMultipleBoundaries(t *testing.T) {
 	}
 	mockSQLExecuter := splitquery_testing.NewMockSQLExecuter(mockCtrl)
 	expectedCall1 := mockSQLExecuter.EXPECT().SQLExecute(
-		"select id, user_id from test_table"+
+		"select id, user_id from test_table force index (`primary`)"+
```

Comment:
It may be better to all-caps PRIMARY, since that is how the index is displayed in show indexes.